examples/local/launch_cloud_onprem.py

```diff
@@ -100,5 +102,5 @@
 print((
     f'Sky onprem cluster {local_cluster_name} is now ready for use! '
     f'You can launch jobs with `sky launch -c {local_cluster_name}  -- [CMD]. '
```

Comment:
nit: there's a double space and missing `:

```
`sky launch -c {local_cluster_name} -- [CMD]`.
```

Reply:
Done, thanks!